core/src/test/java/org/apache/accumulo/core/metadata/schema/TabletMetadataTest.java

```diff
@@ -117,6 +142,15 @@ public void testAllColumns() {
     DIRECTORY_COLUMN.put(mutation, new Value("t-0001757"));
     FLUSH_COLUMN.put(mutation, new Value("6"));
     TIME_COLUMN.put(mutation, new Value("M123456789"));
+    OPID_COLUMN.put(mutation,
+        new Value("SPLITTING:FATE:META:12345678-9abc-def1-2345-6789abcdef12"));
+    SELECTED_COLUMN.put(mutation,
+        new Value(new SelectedFiles(Set.of(new ReferencedTabletFile(
+            new Path("hdfs://nn.somewhere.com:86753/accumulo/tables/42/t-0000/F00001.rf"))
+            .insert()), true, fateId1, SteadyTime.from(100, TimeUnit.NANOSECONDS))
+            .getMetadataValue()));
+    AVAILABILITY_COLUMN.put(mutation, TabletAvailabilityUtil.toValue(TabletAvailability.ONDEMAND));
+    MERGEABILITY_COLUMN.put(mutation, new Value("{\"delay\":1,\"steadyTime\":1,\"never\"=false}"));
```

Comment:
Some of these can be extracted to variables to be used here and in the below assertions to avoid copy-pastes